$OpenBSD: patch-tools_clang_lib_Basic_Targets_cpp,v 1.26 2017/07/14 17:14:03 sthen Exp $

Enable TLS support but default to the emulatated TLS model.

Index: tools/clang/lib/Basic/Targets.cpp
--- tools/clang/lib/Basic/Targets.cpp.orig
+++ tools/clang/lib/Basic/Targets.cpp
@@ -551,8 +551,6 @@ class OpenBSDTargetInfo : public OSTargetInfo<Target> 
 public:
   OpenBSDTargetInfo(const llvm::Triple &Triple, const TargetOptions &Opts)
       : OSTargetInfo<Target>(Triple, Opts) {
-    this->TLSSupported = false;
-
       switch (Triple.getArch()) {
         case llvm::Triple::x86:
         case llvm::Triple::x86_64:
